openedx_lti_tool_plugin/tests/test_signals.py

```diff
@@ -139,20 +142,22 @@ def test_without_lti_profile(
         datetime_mock.now.assert_not_called()
         self.graded_resource.update_score.assert_not_called()
 
+    @ddt.data(None, 0, 0.0)
```

Comment:
If the user scores 0 would't we want to update the grade?

Reply:
@alexjmpb I considered it wasn't necessary but according to our requirements we should also send the update if the score is 0.0, I will modify this condition